src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/test/Configuration/ServiceCollectionExtensionTests.cs

```diff
@@ -66,5 +70,33 @@ public void ClientProxyTest_With_Missing_ProxyURL_ThrowsException()
 
         // Act & Assert
         Assert.Throws<ConfigurationException>(() => serviceCollection.AddWattTimeForecastDataSource(configuration.DataSources()));
+        Assert.Throws<ConfigurationException>(() => serviceCollection.AddWattTimeEmissionsDataSource(configuration.DataSources()));
+    }
+
+    [Test]
+    public void ClientProxyTest_With_Missing_ProxyURL_AddsDataSource()
```

Comment:
Same comments as the one for Electricity Map. Do you know if there is a need for the None Proxy Case?

Reply:
Mainly to increase code coverage for the files (both of them), since this path (use proxy = false) showed up as untested on my code analyzer locally.